server/config/routes.js

```diff
@@ -93,6 +98,41 @@ exports.configure = function (app) {
   app.get('/location/province/:uuid', locations.lookupProvince);
   app.get('/location/detail/:uuid', locations.lookupDetail);
 
+  //API for account routes crud
+  app.get('/accounts', account.list);
+  app.get('/accounts/:id', account.getAccount);
+  app.post('/accounts', account.create);
+  app.put('/accounts/:id', account.update);
+
+  //API for account type routes crud
+  app.get('/account_types', accountType.list);
+  app.get('/account_types/:id', accountType.getAccountType);
+  app.post('/account_types', accountType.create);
+  app.put('/account_types/:id', accountType.update);
+  app.delete('/account_types/:id', accountType.remove);
+
+  //API for cost_center routes crud
+  app.get('/cost_centers', costCenter.list);
+  app.get('/cost_centers/:id', costCenter.getCostCenter);
+  app.post('/cost_centers', costCenter.create);
+  app.put('/cost_centers/:id', costCenter.update);
+  app.delete('/cost_centers/:id', costCenter.remove);
+
+  //API for profit_center routes crud
+  app.get('/profit_centers', profitCenter.list);
+  app.get('/profit_centers/:id', profitCenter.getProfitCenter);
+  app.post('/profit_centers', profitCenter.create);
+  app.put('/profit_centers/:id', profitCenter.update);
+  app.delete('/profit_centers/:id', profitCenter.remove);
+
+  //API for reference routes crud
+  app.get('/references', reference.list);
```

Comment:
It is confusing that the client service `reference` has a function `list()` that calls calls the HTTP URL `/references/default`, but on the server `reference.list` is mapped to the URL `/references`.  Could you make both `list()` methods map to the same URL?